src/containers/SignIn/SignIn.scss

```diff
@@ -0,0 +1,43 @@
+@use "../../assets/sass/variables" as *;
+
+@media only screen and (max-width: $max-mobile-view) {
+  .login-page {
+    font-family: $primary-font-family;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    height: 100vh;
```

Comment:
min-height: -webkit-fill-available;

https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser

Reply:
Thanks for the heads up! I've implemented it now.